src/sentry/uptime/subscriptions/subscriptions.py

```diff
@@ -53,6 +54,7 @@ def create_uptime_subscription(
 
     if created:
         create_remote_uptime_subscription.delay(subscription.id)
+        fetch_subscription_rdap_info.delay(subscription.id)
```

Comment:
I would like for there to be some way for this functionality to actually be mocked out typically in tests. Since this does talk to external APIs.

Right now I think tests actually go and talk to the RDAP RIR services, which is defintiely not what we want.

Is there a way for me to simply auto-mock this in most cases, and turn it off in the cases where we don't want mocks?

Unfortunately the `create_uptime_subscription` function is called in quite a lot of tests, so not sure if the best path forward is to mock it in every place it's used.

Reply:
Tasks don't run unless `with self.tasks` is used in the test.

That said, maybe we can just mock this by default in all of our uptime tests